apps/back-office/src/app/(dashboard)/settings/components/service/AddServiceDialog.tsx

```
@@ -0,0 +1,492 @@
+                    </CommandList>
+                  </Command>
+                </PopoverContent>
+              </Popover>
+            </div>
+            <div className="flex-1">
+              <FormLabel className="text-[#374151] text-[14px] block mb-1.5">
+                Description
+              </FormLabel>
+              <form.Field
+                name="description"
+                children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)